README.md

```diff
@@ -1,45 +1,102 @@
+
+   - Core: This type of proposal includes improvements to the core Hedera Hashgraph protocol and services (cryptocurrency, smart contracts, and file service). 
+
+   - Networking: This type of proposal includes improvements around proposed improvements to network protocol specifications. 
+
+   - Interface: This type of proposal includes improvements around client API/RPC specifications and standards, and also certain language-level standards. 
+
+   - Economics: This type of proposal includes improvements to the Hedera fee structure, token distribution, proof-of-stake, and other areas relating to the economics of Hedera. 
+
+   - HRC: This type of proposal is a Hedera “request for comment” - HIPs under this distinction require elaboration or commentary from the core Hedera team before recategorizing or moving forward with the proposal.
+
+- An Informational HIP - This type of proposal provides general guidelines or information to the Hedera Hashgraph community, but does not propose a new feature. Informational HIPs do not necessarily represent Hedera Hashgraph community consensus or recommendation, and so users are free to ignore Informational HIPs.
+
+## Qualifications for a HIP 
+
+Each HIP should only be one single key proposal and/or idea. The idea should be focused and only issue to one subject matter to be successful. A HIP must meet certain minimum criteria: it must be clear and have a complete description of the proposed enhancement, the enhancement must represent a net improvement, the proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
 
 ## HIP Work Flow
-Parties involved in this process are you, the HIP author, the HIP editors, and the Hashgraph Developers.
 
-Evaluate your idea: Before you begin, ask the Hashgraph community first if your idea is original to save time. Make sure the idea is applicable to the entire community and not to one particular author. While it may be a good idea for an author it may not be the case for all areas of the Hashgraph community. An excellent place to discuss your proposal is here in the Issues section of this repository. You can start creating a more formalized language around your HIP.
+Parties involved in this process are you (the HIP author), the HIP editors, and the Hedera Governing Council (Hedera Hashgraph LLC). 
+
+#### The following is the process that a successful HIP will move along: 
+
+   - Work in Progress (WIP) 
+
+   - Draft 
+
+   - Last Call 
+
+   - Accepted by the community 
+
+   - Proposed to the Hedera Governing Council 
+
+   - Accepted by the Hedera Governing Council 
+
+   - Final 
 
-Following is the process that a successful EIP will move along:
+[ WIP ] -> [ DRAFT ] -> [ LAST CALL ] -> [ ACCEPTED BY THE COMMUNITY ] -> [ PROPOSED TO HGC ] -> [ ACCEPTED BY HGC ] -> [ FINAL ] 
```

Comment:
I think this information duplicates the bullet list above it and can probably be removed. Maybe we should just number the above list instead?

Reply:
I think it was added as a redundancy. When it was shorter, it was more easy to understand the process which it goes along. Since we have additional steps and isn't as clear, I would agree it should be left out and number the list above 👍